applications/openshift/etcd/etcd_check_cipher_suite/rule.yml

```diff
@@ -0,0 +1,41 @@
+prodtype: ocp4
+
+title: Ensure ETCD has correct cipher suite
+
+description: |-
+  Check the current cipher suite used in ETCD.
+
+rationale: |-
+  Use of weak or untested encryption algorithms undermines the purposes of utilizing encryption to
+  protect data. The system must implement cryptographic modules adhering to the higher
+  standards approved by the federal government since this provides assurance they have been tested
+  and validated.
+
+
+identifiers:
+  cce@ocp4: CCE-85866-2
+
+references:
+  pcidss: Req-4.1
+
+ocil_clause: 'ETCD cipher suite does not meet requiriements'
```

Comment:
s/requiriements/requirements

Reply:
Thanks @Vincent056, spelling is my struggle